### PR TITLE
WINUI-843320 Prepared the Built-in Types UG doument content for Shimmer control in WinUI. 

### PR DESCRIPTION
**Pull request by GilbertmanojSF3971**
Prepared the Built-in Types UG doument content for Shimmer control in WinUI

### CODE REVIEW

File: winui/Shimmer/Built-in-Types.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: Built-in Types of WinUI Shimmer Control | Syncfusion
+description: Learn about the various built-in types available in the Syncfusion WinUI Shimmer(SfShimmer) control, including Circle Persona, Square Persona, Profile, Article, Video, Feed, and Shopping.
+platform: WinUI
```
Comment (Manivannan-E) [inline winui/Shimmer/Built-in-Types.md:5]: @GilbertmanojSF3971 

title: Built-in types of WinUI Shimmer control | Syncfusion
description: Learn about the various built-in types available in the Syncfusion WinUI Shimmer (SfShimmer) control, including CirclePersona, SquarePersona, Profile, Article, Video, Feed, and Shopping.

File: winui/Shimmer/Built-in-Types.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+---
+
+# Built-in Types of WinUI Shimmer Control (SfShimmer)
+
+Syncfusion WinUI Shimmer (SfShimmer) control provides seven built-in shimmer view types such as CirclePersona, SquarePersona, Profile, Article, Video, Feed, and Shopping.
+
+![WinUI Shimmmer control with DefaultView](Shimmer_images/winui_shimmer_BuiltinView.gif)
+
+The built-in shimmer types can be used by setting the Type of SfShimmer. By default, the CirclePersona is initially rendered.
+
```
Comment (Manivannan-E) [inline winui/Shimmer/Built-in-Types.md:17]: @GilbertmanojSF3971 

The Syncfusion WinUI Shimmer (SfShimmer) control offers a range of built-in shimmer types, each designed to serve specific user interface needs. These built-in types provide convenient options for creating captivating loading animations in your WinUI applications. The following built-in shimmer types such as CirclePersona, SquarePersona, Profile, Article, Video, Feed, and Shopping is available in Shimmer control.

WinUI Shimmmer control with DefaultView

By default, the WinUI Shimmer control initializes with the `CirclePersona` type. You can easily switch between these built-in types by setting the `Type` property of the SfShimmer control.

File: winui/Shimmer/Built-in-Types.md (hunk 1 above)
Comment (GilbertmanojSF3971) [reply winui/Shimmer/Built-in-Types.md:5]: @Manivannan-E Sir. Changed the title and description content.

File: winui/Shimmer/Built-in-Types.md (hunk 2 above)
Comment (GilbertmanojSF3971) [reply winui/Shimmer/Built-in-Types.md:17]: @Manivannan-E Sir , Modified the content of the details about built-in types

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

**Pull request merged**